client/issue.go

```diff
@@ -42,6 +46,12 @@ func (c *Client) GetMyIssues() ([]Issue, error) {
 	return c.GetIssues("assigned_to_id=me")
 }
 
+// GetMyIssues fetches issues assigned only to currently logged user.
+func (c *Client) GetMyIssuesOnly() ([]Issue, error) {
+	param := fmt.Sprintf("assigned_to_id=%v", viper.GetString(config.UserID))
```

Comment:
Isn't this exactly the same as `c.GetIssues("assigned_to_id=me")`?

Reply:
Flag "me" shows you every task that is assigned to you,as well as evey group you are part of.
So,if user participate in group of Development and Maintenance, flag "me" will return every task  that is assigned to these groups as well.
If you pass an id of user,you will get tasks only assigned to you,and task assigned to group(s) will be excluded.

Reply:
Maybe next link could provide more explanation.
https://www.redmine.org/issues/13758